### PR TITLE
Validate RODAI

### PR DESCRIPTION
**Pull request by llegoelkelo**
# Validate [RODAI](https://solscan.io/token/GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/RodAISol/status/1755262566366945630
- Coingecko/ CMC URL (If available): https://coinmarketcap.com/currencies/rod-ai/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by llegoelkelo** [comment]
@Jymfletzfur resubmitting our PR (previous #1081) now that we have reasonable volume and mcap.

**Comment by llegoelkelo** [comment]
Hey @9yoi / @Jymfletzfur / @0xSoju2, could you please check us? We waited over a month on #1081.

We now have adequate MC and volume plus an ever-growing list of holders.

In addition to this, we are amongst the top gainers this week on Solana on CMC.

We are a reputable project and we are doing this for our community. Thank you in advance.

Birdeye: https://birdeye.so/token/GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC?chain=solana
Rugcheck: https://rugcheck.xyz/tokens/GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC

IMG_0117

**Comment by llegoelkelo** [comment]
today we hit 1M market cap 🚀 🏆 💎

**Comment by llegoelkelo** [comment]
Let me kick it off.

ROD.AI project is community based. I can confirm that the provided token information is accurate. The token is valid.

Mint address: `GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC`
Symbol: `RODAI`
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by RojoCryptonian** [comment]
I've being part of this project since the beginning and I asure this is one of the strongest project in the Solana ecosystem in a long time. The community is continuously working everyday to make RODAI not only as a memecoin, but a tool for everyone.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by Dejps1** [comment]
I have been in this project since 100k marketcap and all the way to here 1milion market cap. I decided to stay with the project this long because of the loyal team that stands behind the project, because they work not only on their own success, but above all, they fight for the success of others.  the project is on good track.


Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

File: validated-tokens.csv
Comment (Agico67) [inline validated-tokens.csv:1]: ❗️JUPITER GITHUB ATTESTATIONS NEEDED❗️
Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Review by Agico67 — COMMENTED** [review]
Urgent: Need GitHub Attestations for $RODAI
I'm reaching out to urgently request your GitHub attestations for $RODAI. Your support is crucial for the success and transparency of our project. Thank you in advance for your valuable contribution!
Best regards,
Albert

**Comment by ValenciaVZ** [comment]
ROD.AI is a solid project and legitimate community. I attest that the information below is correct.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by LordEcneralc** [comment]
I’ve only found them recently, a friend of mine researched them and told me they looked like a good project to get in on. After joining their community I’ve seen how passionate the devs and community are about this project. This is a valid token and I’m in for the long haul.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by CadaCrypto** [comment]
This is the first time I hold in a project like this. All info they provided is correct.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by MorrocoyBoy** [comment]
ROD.AI is a valid project with an incredible community. I have been part of the community since early February and I can attest that the information below is accurate:

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by ushiland** [comment]
I joined the ROD.AI community 3 weeks ago and I love this project! I can attest that the information below is accurate and that our project and community are legitimate.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by SantiSolvers** [comment]
Developers are transparent with al the info and with their plans for RODAI's future. After reading their Whitepaper and Roadmap I can tell all the info is correct.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by CoinDevotion** [comment]
Our community and project ROD.AI are as legitimate as can be. I've been part of the community for over a month and I can attest that the information below is indeed correct.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by Botagent544** [comment]
I have been a part of the project for almost 2 months, the efforts of the team and the project has grown time folds in that time. The community is now expanding to make this project an integral part and necessity to the Solana's ecosystem to expand oncoming coins into the ecosystem.   

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by CryptoniansXYZ** [comment]
I've been in this project since day 1. We have found a committed and loyal community attracted to the transparency and potencial RODAI has. The info provided is 100% correct.

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list.

**Comment by FreeAgent37** [comment]
The RodAi team is working very hard and the token is completely valid and is not a scam coin. One of the best projects to date. 

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by maestrow17** [comment]
This is a genuine project. The developers are working hard to push it forward and bring innovation to the space and the Solana blockchain. I can attest that they are legit. Keep up the good work!

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by ccqueen86** [comment]
As of the 8th March 2024, I joined the RODAI project. Prior to joining, I did my research by examining what persons were saying about the project via Twitter. Also, when I looked at the team's website, all the information was laid out clearly. Furthermore, I was able to have my questions answered and communicate with the development team via their Telegram group and social media handles. For quite some time, I had been looking for a legitimate project that I could invest and support. RODAI is that project. I can attest that it is legitimate and is not a scam. The development team is always present. The community is expanding and is highly supportive of the project. I know that what they are building will not only bring further utility to the Solana blockchain, but would help other projects in the space. 

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by Vitaliks001** [comment]
$RODAI is absolutely killing it! The gains have been off the charts and the RODkit launchpad is a game-changer. Keep up the awesome work! 🚀💪@jupiterexchange list $RODAI


Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png

**Comment by skullrichie** [comment]
$RODAI is a really good and genuine project, every member of the team are working so hard to push the project forward, this is a must list project @JupiterExchange

Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by Bullish-runs** [comment]
I’m part of the $rodai community this is a evergrowing project who cares for their community, loves to answer any concerns the token mint address is indeed the correct one and has real future plans for the community.
Mint address: GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC
Symbol: RODAI
Token Decimals: 5
Token icon: https://www.rodai.meme/logo.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by llegoelkelo** [comment]
Hey @9yoi / @Jymfletzfur / @0xSoju2 our PR got closed when I synced the fork and this commit is not not getting pulled in: https://github.com/rodaisol/token-list/commit/26a8f119ca55bda7ca983dd76a8a62a7e6575fce

**Review by Jymfletzfur — APPROVED** [review]

**Comment by llegoelkelo** [comment]
Hey @9yoi / @Jymfletzfur / @0xSoju2, please let me know if there is anything else needed here.

**Comment by 9yoi** [comment]
Moved here because I couldn't resolve conflicts on this PR. Will approve and merge #1892 https://github.com/jup-ag/token-list/pull/1892

**Comment by llegoelkelo** [comment]
thank you @9yoi

**Pull request closed without merging**